src/audio/kpb.c

```diff
@@ -1627,6 +1668,24 @@ static void kpb_drain_samples(void *source, struct audio_stream __sparse_cache *
 	}
 }
 
+void kpb_convert_32b_to_24b(const struct audio_stream __sparse_cache *source, int ioffset,
```

Comment:
Consider declaring this function as static if it is not going to be used outside the module.